### DISCUSSION TITLE
Constant potential chemistry with DP for the full battery Capacitors?

### DISCUSSION THREAD

**Discussion by ms123456-TX** [deepmodeling/deepmd-kit#2080] [Q&A]
Is it possible for DP to make a complete battery system and study its chemistry. For example, with electrodes on 2 sides and electrolyte in the middle, and with positive and negative electrodes at constant potential?

**Comment by HuangJiameng** [comment]
Just a kindly reminder, you can raise a feature request if it's necessary. https://github.com/deepmodeling/deepmd-kit/issues/new?assignees=&labels=enhancement&template=feature-request.yml&title=%5BFeature+Request%5D+_Replace+with+Title_
